packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/optimize-ciemss-operation.ts

```diff
@@ -66,6 +66,7 @@ export interface OptimizeCiemssOperationState extends BaseState {
 	inProgressPostForecastId: string;
 	postForecastRunId: string;
 	optimizationRunId: string;
+	optimizedInterventionPolicy: InterventionPolicy | null;
```

Comment:
Can you comment on roughly how large this `optimizedInterventionPolicy` is on average? 

We generally try to minimize the amount of actual data stored in the workflow itself, because the workflow object can get expensive to fetch and to update.

Reply:
I can't comment how large it would be on average but this would be an example object with a few interventions.  I _could_ look into just storing the intervention policy id on the state and fetch the object if its present instead

```
{
    "id": "99c46fcd-b482-4b46-84d4-6942614e11b0",
    "createdOn": "2024-07-19T21:50:05.284+00:00",
    "updatedOn": "2024-07-19T21:50:05.284+00:00",
    "name": "New Intervention Policy",
    "description": "This is a new intervention policy.",
    "fileNames": [],
    "temporary": false,
    "publicAsset": false,
    "modelId": "35486f82-9675-44ec-95c4-6fc4e0fb312c",
    "interventions": [
        {
            "name": "Toronto",
            "appliedTo": "β",
            "type": "parameter",
            "staticInterventions": [
                {
                    "timestep": 30,
                    "value": 0.4
                },
                {
                    "timestep": 20,
                    "value": 0.5
                },
                {
                    "timestep": 10,
                    "value": 0.7
                }
            ],
            "dynamicInterventions": []
        },
        {
            "name": "New York",
            "appliedTo": "β",
            "type": "parameter",
            "staticInterventions": [
                {
                    "timestep": 67,
                    "value": 0.8
                }
            ],
            "dynamicInterventions": []
        },
        {
            "name": "Chicago",
            "appliedTo": "γ",
            "type": "parameter",
            "staticInterventions": [
                {
                    "timestep": 16,
                    "value": 0.6
                },
                {
                    "timestep": 13,
                    "value": 0.2
                },
                {
                    "timestep": 58,
                    "value": 0.1
                }
            ],
            "dynamicInterventions": []
        },
        {
            "name": "Lisbon",
            "appliedTo": "γ",
            "type": "parameter",
            "staticInterventions": [
                {
                    "timestep": 45,
                    "value": 0.4
                }
            ],
            "dynamicInterventions": []
        }
    ]
}
```

Reply:
Cool Ill merge this in and then make that change